packages/changed-elements-react/src/api/ReportGenerator.ts

```diff
@@ -435,11 +435,19 @@ export class ReportGenerator extends ReportGeneratorBase {
     entries: ChangedElementEntry[],
     isNew?: boolean,
   ): Promise<void> => {
-    const rulesetOrId = "Default";
+    // We cannot rely on the "Default" backend ruleset because there is a bug in Presentation library
+    // where invalid ECSQL query is generated during getContent RPC call for rulesets as simple as
```

Comment:
Have we created an issue on their side or followed up to make sure it gets solved on their end as well?

Reply:
I haven't got around to it yet as I need to create a minimal reproduction for them.